app.js

```diff
@@ -245,6 +245,92 @@ app.post('/upload', function(req, res) {
 	});
 });
 
+app.post('/processTimetable', function(req, res) {
```

Comment:
Can you make this so that it only returns the ics file to the user? I want to re-write the email part later on so remove everything but the return of the ics file.